drf_user/models.py

```diff
@@ -35,11 +35,14 @@ class User(AbstractBaseUser, PermissionsMixin):
     username = models.CharField(
         verbose_name=_("Unique UserName"), max_length=254, unique=True
     )
-    email = models.EmailField(verbose_name=_("EMail Address"), unique=True)
+    email = models.EmailField(verbose_name=_("Email Address"), unique=True)
     mobile = models.CharField(
         verbose_name=_("Mobile Number"), max_length=150, unique=True
     )
     name = models.CharField(verbose_name=_("Full Name"), max_length=500, blank=False)
+    profile_image = models.ImageField(
+        verbose_name=_("Profile Photo"), null=True, blank=True
```

Comment:
Could you please add an upload path here, otherwise uploaded images will be stored in the root of the MEDIA folder. For example, you can name the path like this, `upload_to='user_images/`.